test/functional/apps/discover/_discover.js

```diff
@@ -0,0 +1,134 @@
+define(function (require) {
+  var Common = require('../../../support/pages/Common');
+  var SettingsPage = require('../../../support/pages/SettingsPage');
+  var DiscoverPage = require('../../../support/pages/DiscoverPage');
+  var expect = require('intern/dojo/node!expect.js');
+
+  return function (bdd, scenarioManager) {
+    bdd.describe('discover app', function describeIndexTests() {
+      var common;
+      var settingsPage;
+      var discoverPage;
+      var remote;
+      var fromTime;
+      var toTime;
+      this.timeout = 3 * 60 * 1000;
```

Comment:
There's a lot of very large timeouts specified in a variety of places in these tests. What's causing them to run so slow?